Makefile

```diff
@@ -231,7 +267,7 @@ build/assets/text/nes_message_data_static.o: build/assets/text/message_data.enc.
 build/assets/text/staff_message_data_static.o: build/assets/text/message_data_staff.enc.h
 
 build/assets/%.o: assets/%.c
-	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $<
+	$(CC) -c $(CFLAGS) -I $(<D) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $(PIPEIN)
```

Comment:
What's the motivation of adding `-I $(<D)`?

Reply:
GCC will error being unable to find file_choose.h if you don't.

Reply:
why that one specifically, do you know?

Reply:
Cant reproduce it actually. Swear I was seeing this earlier, even during GCC research. Removing these now.